notebooks/hyperkitty_to_csv.ipynb

```diff
@@ -0,0 +1,841 @@
+{
```

Comment:
<p>Can you rerun this cell so that it does not contain the error output? I tried to reproduce it but could not. I did find changing the year of interest from 2006 to 2010 made it run quickly without issue whereas 2006 seemed to hang indefinitely. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/fedora-mailing-list-analysis/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>